src/main/java/wooteco/subway/line/domain/Section.java

```diff
@@ -36,6 +36,14 @@ public Station getDownStation() {
         return downStation;
     }
 
+    public Long getUpStationId(){
+        return upStation.getId();
+    }
+
+    public Long getDownStationId(){
+        return downStation.getId();
+    }
```

Comment:
```suggestion
    public Long getUpStationId() {
        return upStation.getId();
    }

    public Long getDownStationId() {
        return downStation.getId();
    }
```

<img width="379" alt="스크린샷 2021-05-22 오전 12 20 45" src="https://user-images.githubusercontent.com/18049492/119160860-a704ca80-ba93-11eb-9827-e5608b67e4a6.png">


단축키 사용을 습관하거나, 인텔리제이에서 커밋하신다면 Reformat code 옵션을 켜는 건 어떨까요?